idb/idb.go

```diff
@@ -334,6 +334,10 @@ type ApplicationRow struct {
 // IndexerDbOptions are the options common to all indexer backends.
 type IndexerDbOptions struct {
 	ReadOnly bool
+	// Maximum connection number for connection pool
+	MaxConns int32
+	// Minimum connection number for connection pool
+	MinConns int32
```

Comment:
This is the better approach, but it looks like you didn't use it.

What is the use case for `MinConns`, do we need to set it?